backend/de.metas.swat/de.metas.swat.base/src/test/java/de/metas/invoicecandidate/C_Invoice_Candidate_Builder.java

```diff
@@ -459,6 +461,12 @@ public C_Invoice_Candidate_Builder setDateInvoiced(final LocalDate dateInvoiced)
 		return this;
 	}
 
+	public C_Invoice_Candidate_Builder setDateToInvoice(final LocalDate dateToInvoice)
```

Comment:
pls clarify why setDateToInvoice is relevant for current tests?

Reply:
The default rule to compute is BASELINETYPE.InvoicedDate. SO when computing baseline date we have available only dateToInvoice and not dateinvoiced, yet!